nixos/modules/services/databases/redis.nix

```diff
@@ -46,16 +46,12 @@ in
 
     services.redis = {
 
-      enable = mkOption {
-        type = types.bool;
-        default = false;
-        description = ''
-          Whether to enable the Redis server. Note that the NixOS module for
-          Redis disables kernel support for Transparent Huge Pages (THP),
-          because this features causes major performance problems for Redis,
-          e.g. (https://redis.io/topics/latency).
-        '';
-      };
+      enable = mkEnableOption ''
+        Whether to enable the Redis server. Note that the NixOS module for
```

Comment:
`mkEnableOption`'s description is basically `Whether to enable <name>`, while `<name>` is the first argument passed to `mkEnableOption`. Thereforce I think that your text might seem rather confusing when looking at it from the manual. I'd suggest that you either adjust your text or that you do e.g. `(mkEnableOption "opt-name") // { description = ''A little more details''; }`.